main.js

```diff
@@ -0,0 +1,217 @@
+var starWarsObj = function() {
+  this.getPerson = person;
+  this.getPeople = people;
+  this.getFilm = film;
+  this.getFilms = films;
+  this.getPlanet = planet;
+  this.getPlanets = planets;
+  this.getSpecies = species;
+  this.getAllSpecies = allSpecies;
+  this.getStarship = starship;
+  this.GetStarships = starships;
+  this.getVehicle = vehicle;
+  this.getVehicles = vehicles;
```

Comment:
these are actually irrelevant as they will all be undefined (try starWarsObj.getPerson in the console to see what I mean). You could delete lines 1-13 and it would work exactly the same.